CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Pkg v1.9 Release Notes
   or `pkg> add https://github.com/Org/Package.jl/commit/bb9eb77e6dc`.
 - Timing of the precompilation of dependencies can now be reported via `Pkg.precompile(timing=true)` ([#3334])
 - Bug fix on `pin/free --all` which now correctly applies to all dependencies, not just direct dependencies ([#3346]).
+- New `add --installed Foo` or `Pkg.add("Foo", allow_max=true)` flag to allow the tiered resolver to first try to
```

Comment:
I get why these aren't named the same, but it's a predictable source of confusion.

Reply:
Yeah. I've made them both `installed` now